### PR TITLE
glew 1.12.0

### PR DESCRIPTION
**Pull request by ortho**
update to glew 1.12.0

### CODE REVIEW

File: Library/Formula/glew.rb [hunk 1]
```diff
@@ -30,4 +20,8 @@ def install
     system "make", "GLEW_PREFIX=#{prefix}", "GLEW_DEST=#{prefix}", "all"
     system "make", "GLEW_PREFIX=#{prefix}", "GLEW_DEST=#{prefix}", "install.all"
   end
+
+  test do
+    assert_equal "GLEW version 1.12.0", shell_output("#{bin}/glewinfo | grep \"GLEW version\"").strip
```
Comment (DomT4) [inline Library/Formula/glew.rb]: Let's not make this version specific. You can also just do:

`assert_match /#{version}/, shell_output("#{bin}/glewinfo")`

Which will do the same thing.

**Comment by jacknagel** [comment]
Probably need to look at bumping revisions on dependents here, as it looks like the library name has changed.

**Comment by ortho** [comment]
Do I need to check all package that uses glew library?

**Comment by jacknagel** [comment]
```
$ brew uses glew -r
arx-libertatis	csfml		gource		libdrawtext	logstalgia	mal4s		openclonk	sfml
```

These need to be checked for dynamic linkage and have their revisions bumped accordingly.

**Comment by ortho** [comment]
I've just tested everything is working fine.
According to the relese note there are only fixes minor bugs and adds new OpenGL extensions

I don't know what I have to do with openimageio package.
Do I have to pull request to homebrew-science?

**Comment by jacknagel** [comment]
You'll need to add `revision 1` to each of the dependents, and leave the bottle blocks intact (they will be automatically updated).

**Comment by xu-cheng** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Pull request closed without merging**